File: BackEndAPI/Data/Entities/HopDong.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace BackEndAPI.Entities
{
    public partial class HopDong
    {
        public int MaNguoiDung { get; set; }
        public int MaHopDong { get; set; }
        public DateTime? NgayDangKy { get; set; }
        public DateTime? NgayKyHopDong { get; set; }
        public DateTime? NgayHieuLuc { get; set; }
        public DateTime? NgayKetThuc { get; set; }
        public string GiayPhepKinhDoanhImg { get; set; }
        public string ChungNhanAnToanImg { get; set; }
        public string HopDongImg { get; set; }
        public virtual NguoiDung NguoiDung { get; set; }
    }
}

File: BackEndAPI.Java/DiChoThue/src/main/java/com/example/DiChoThue/Entities/HopDong.java
package com.example.DiChoThue.Entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "HOP_DONG")
public class HopDong {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int ma_hop_dong;
    private int ma_nguoi_dung;
    private Date ngay_ky_hop_dong;
    private Date ngay_hieu_luc;
    private Date ngay_ket_thuc;
    private String chungNhanAnToanImg;
    private String giayPhepKinhDoanhImg;
    private String hopDongImg;
    private Date ngayDangKy;
}
